core/templates/dev/head/domain/learner_dashboard/learner_dashboard_icons_directive.html

```diff
@@ -19,7 +19,15 @@
    uib-tooltip="<['I18N_LIBRARY_ACTIVITY_COMPLETED_ICON' | translate]>"
    tooltip-placement="left"></i>
 <i class="oppia-learner-dashboard-icon fa fa-spinner"
-   ng-if="belongsToIncompleteActivities()"
+   ng-if="true"
```

Comment:
Why does the 'ng-if' need to be true always? Same as above, by setting it to true, shows the spin symbol for all the activities (which is actually for those activities which belong to incomplete activities).

Reply:
@srikar0896 you didn't respond to this?

Reply:
@seanlip yes I have replied in the above message,as they are both related to same issue.

Reply:
ah ok. In general please make sure that every message is resolved, otherwise it's hard to tell! (You could say "see previous comment", for example.) Thanks for clarifying; merging now.